 Network cristiano182_default  Creating
 Network cristiano182_default  Created
 Container cristiano182-postgres-1  Creating
 Container cristiano182-postgres-1  Created
 Container cristiano182-api02-1  Creating
 Container cristiano182-api01-1  Creating
 Container cristiano182-api01-1  Created
 Container cristiano182-api02-1  Created
 Container cristiano182-nginx-1  Creating
 Container cristiano182-nginx-1  Created
Attaching to api01-1, api02-1, nginx-1, postgres-1
postgres-1  | The files belonging to this database system will be owned by user "postgres".
postgres-1  | This user must also own the server process.
postgres-1  | 
postgres-1  | The database cluster will be initialized with locale "en_US.utf8".
postgres-1  | The default database encoding has accordingly been set to "UTF8".
postgres-1  | The default text search configuration will be set to "english".
postgres-1  | 
postgres-1  | Data page checksums are disabled.
postgres-1  | 
postgres-1  | fixing permissions on existing directory /var/lib/postgresql/data ... ok
postgres-1  | creating subdirectories ... ok
postgres-1  | selecting dynamic shared memory implementation ... posix
postgres-1  | selecting default max_connections ... 100
postgres-1  | selecting default shared_buffers ... 128MB
postgres-1  | selecting default time zone ... UTC
postgres-1  | creating configuration files ... ok
postgres-1  | running bootstrap script ... ok
nginx-1     | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx-1     | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx-1     | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx-1     | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
nginx-1     | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx-1     | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx-1     | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx-1     | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx-1     | /docker-entrypoint.sh: Configuration complete; ready for start up
postgres-1  | sh: locale: not found
postgres-1  | 2024-02-19 03:21:03.130 UTC [30] WARNING:  no usable system locales were found
api01-1     | yarn run v1.22.21
api02-1     | yarn run v1.22.21
api01-1     | $ node dist/infra/server/Server.js
api02-1     | $ node dist/infra/server/Server.js
postgres-1  | performing post-bootstrap initialization ... ok
api02-1     | /app/node_modules/pg-pool/index.js:45
api02-1     |     Error.captureStackTrace(err);
api02-1     |           ^
api02-1     | 
api02-1     | Error: connect ECONNREFUSED 172.27.0.2:5432
api02-1     |     at /app/node_modules/pg-pool/index.js:45:11
api02-1     |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
api02-1     |   errno: -111,
api02-1     |   code: 'ECONNREFUSED',
api02-1     |   syscall: 'connect',
api02-1     |   address: '172.27.0.2',
api02-1     |   port: 5432
api02-1     | }
api02-1     | 
api02-1     | Node.js v20.11.0
api01-1     | /app/node_modules/pg-pool/index.js:45
api01-1     |     Error.captureStackTrace(err);
api01-1     |           ^
api01-1     | 
api01-1     | Error: connect ECONNREFUSED 172.27.0.2:5432
api01-1     |     at /app/node_modules/pg-pool/index.js:45:11
api01-1     |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
api01-1     |   errno: -111,
api01-1     |   code: 'ECONNREFUSED',
api01-1     |   syscall: 'connect',
api01-1     |   address: '172.27.0.2',
api01-1     |   port: 5432
api01-1     | }
api01-1     | 
api01-1     | Node.js v20.11.0
api02-1     | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
api02-1     | error Command failed with exit code 1.
api01-1     | error Command failed with exit code 1.
api01-1     | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
postgres-1  | syncing data to disk ... ok
postgres-1  | 
postgres-1  | 
postgres-1  | Success. You can now start the database server using:
postgres-1  | 
postgres-1  |     pg_ctl -D /var/lib/postgresql/data -l logfile start
postgres-1  | 
postgres-1  | initdb: warning: enabling "trust" authentication for local connections
postgres-1  | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
postgres-1  | waiting for server to start....2024-02-19 03:21:05.343 UTC [36] LOG:  starting PostgreSQL 15.6 on x86_64-pc-linux-musl, compiled by gcc (Alpine 12.2.1_git20220924-r10) 12.2.1 20220924, 64-bit
postgres-1  | 2024-02-19 03:21:05.347 UTC [36] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
postgres-1  | 2024-02-19 03:21:05.356 UTC [39] LOG:  database system was shut down at 2024-02-19 03:21:04 UTC
postgres-1  | 2024-02-19 03:21:05.382 UTC [36] LOG:  database system is ready to accept connections
postgres-1  |  done
postgres-1  | server started
postgres-1  | CREATE DATABASE
postgres-1  | 
postgres-1  | 
postgres-1  | /usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*
postgres-1  | 
postgres-1  | waiting for server to shut down....2024-02-19 03:21:06.218 UTC [36] LOG:  received fast shutdown request
postgres-1  | 2024-02-19 03:21:06.250 UTC [36] LOG:  aborting any active transactions
postgres-1  | 2024-02-19 03:21:06.252 UTC [36] LOG:  background worker "logical replication launcher" (PID 42) exited with exit code 1
postgres-1  | 2024-02-19 03:21:06.252 UTC [37] LOG:  shutting down
postgres-1  | 2024-02-19 03:21:06.256 UTC [37] LOG:  checkpoint starting: shutdown immediate
postgres-1  | 2024-02-19 03:21:06.755 UTC [37] LOG:  checkpoint complete: wrote 918 buffers (2.8%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.378 s, sync=0.107 s, total=0.504 s; sync files=301, longest=0.083 s, average=0.001 s; distance=4222 kB, estimate=4222 kB
postgres-1  | 2024-02-19 03:21:06.761 UTC [36] LOG:  database system is shut down
postgres-1  |  done
postgres-1  | server stopped
postgres-1  | 
postgres-1  | PostgreSQL init process complete; ready for start up.
postgres-1  | 
postgres-1  | 2024-02-19 03:21:06.846 UTC [1] LOG:  starting PostgreSQL 15.6 on x86_64-pc-linux-musl, compiled by gcc (Alpine 12.2.1_git20220924-r10) 12.2.1 20220924, 64-bit
postgres-1  | 2024-02-19 03:21:06.846 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
postgres-1  | 2024-02-19 03:21:06.846 UTC [1] LOG:  listening on IPv6 address "::", port 5432
postgres-1  | 2024-02-19 03:21:06.851 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
postgres-1  | 2024-02-19 03:21:06.858 UTC [52] LOG:  database system was shut down at 2024-02-19 03:21:06 UTC
postgres-1  | 2024-02-19 03:21:06.869 UTC [1] LOG:  database system is ready to accept connections
api01-1 exited with code 0
api02-1 exited with code 0
api01-1     | yarn run v1.22.21
api01-1     | $ node dist/infra/server/Server.js
api02-1     | yarn run v1.22.21
api02-1     | $ node dist/infra/server/Server.js
nginx-1     | 2024/02/19 03:21:07 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.27.0.3:3000/clientes/1/extrato", host: "localhost:9999"
nginx-1     | 2024/02/19 03:21:07 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.27.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.27.0.4:3001/clientes/1/extrato", host: "localhost:9999"
api01-1     | Postgres Connected: postgres://admin:123@postgres:5432/rinha and load initial script
api01-1     | Server listen on 3000 PORT
api02-1     | Server listen on 3001 PORT
nginx-1     | 2024/02/19 03:21:09 [error] 29#29: *4 no live upstreams while connecting to upstream, client: 172.27.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1     | 2024/02/19 03:21:11 [error] 29#29: *5 no live upstreams while connecting to upstream, client: 172.27.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1     | 2024/02/19 03:21:13 [error] 29#29: *6 no live upstreams while connecting to upstream, client: 172.27.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1     | 2024/02/19 03:21:15 [error] 29#29: *7 no live upstreams while connecting to upstream, client: 172.27.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1     | 2024/02/19 03:21:17 [error] 29#29: *8 no live upstreams while connecting to upstream, client: 172.27.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1 exited with code 0
nginx-1 exited with code 0
api01-1 exited with code 0
api01-1 exited with code 1
api02-1 exited with code 0
api02-1 exited with code 1
postgres-1  | 2024-02-19 03:25:34.449 UTC [1] LOG:  received fast shutdown request
postgres-1  | 2024-02-19 03:25:34.455 UTC [1] LOG:  aborting any active transactions
postgres-1  | 2024-02-19 03:25:34.457 UTC [1] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
postgres-1  | 2024-02-19 03:25:34.457 UTC [50] LOG:  shutting down
postgres-1  | 2024-02-19 03:25:34.460 UTC [50] LOG:  checkpoint starting: shutdown immediate
postgres-1  | 2024-02-19 03:25:34.714 UTC [50] LOG:  checkpoint complete: wrote 719 buffers (2.2%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.083 s, sync=0.156 s, total=0.258 s; sync files=83, longest=0.122 s, average=0.002 s; distance=2376 kB, estimate=2376 kB
postgres-1  | 2024-02-19 03:25:34.719 UTC [1] LOG:  database system is shut down
postgres-1 exited with code 0
